### ISSUE TITLE
Fix Dependency Errors

### ISSUE THREAD

**Issue by Rishi556** [3speaknetwork/video-encoder#30]
There's some peer dependency issues causing `--force` to be used with npm. These should be fixed.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: graphql-type-long@0.1.1
npm ERR! Found: graphql@15.8.0
npm ERR! node_modules/graphql
npm ERR!   graphql@"^15.8.0" from the root project
npm ERR!   peer graphql@"^14.7.0 || ^15.3.0" from express-graphql@0.12.0
npm ERR!   node_modules/express-graphql
npm ERR!     express-graphql@"^0.12.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer graphql@"^0.9.1 || ^0.10.0 || ^0.12.0 || ^0.13.0" from graphql-type-long@0.1.1
npm ERR! node_modules/graphql-type-long
npm ERR!   graphql-type-long@"^0.1.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: graphql@0.13.2
npm ERR! node_modules/graphql
npm ERR!   peer graphql@"^0.9.1 || ^0.10.0 || ^0.12.0 || ^0.13.0" from graphql-type-long@0.1.1
npm ERR!   node_modules/graphql-type-long
npm ERR!     graphql-type-long@"^0.1.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by vaultec81** [comment]
I'm planning to replace express-graphql entirely as it's deprecated. Stay tuned..

**Comment by vaultec81** [comment]
Should be fixed by 6ef3b66c9f7db6c0fa7b87c49a7aa3544304fd8d 6f7c758378170f0b20e9ed2640ffdfac5bfa27b1

Let me know if you encounter this issue again
